src/content/tools/devtools/extensions.md

```diff
@@ -30,20 +34,29 @@ file in the root of the user's project
 This file stores per-project
 (or optionally, per user) settings for DevTools.
```

Comment:
**nitpick:** This looks to be more of a default vs. option rather than noting it as an optional setting.

```suggestion
This file stores DevTools settings either per project (default)
or per user.
How the file stores settings depends upon whether
the file is checked into source control.
```